WaterNetworkManagement/Simulation/SimulationScenario/doc/spec.md

```diff
@@ -0,0 +1,229 @@
+    -   Attribute type: `Property`.Text
+    -   Optional
+
+-   duration: A free text description
+    -   Attribute type: `Property`.Text
+    -   Optional
+
+-   hydraulicTimeStep: A free text description
+    -   Attribute type: `Property`.Text
+    -   Optional
+
+-   flowUnits: A free text description
+    -   Attribute type: `Property`.Text
+    -   Optional
+
+-   headlossFormula: A free text description
+    -   Attribute type: `Property`.Text
+    -   Optional
+
+-   patternStep: A free text description
+    -   Attribute type: `Property`.Text
+    -   Optional
+
+-   patternStart: A free text description
+    -   Attribute type: `Property`.Text
+    -   Optional
+
+-   reportStep: A free text description
+    -   Attribute type: `Property`.Text
+    -   Optional
+
+-   reportStart: A free text description
+    -   Attribute type: `Property`.Text
+    -   Optional
+
+-   ruleStep: A free text description
+    -   Attribute type: `Property`.Text
+    -   Optional
+
+-   statistic: A free text description
+    -   Attribute type: `Property`.Text
+    -   Optional
+
+-   trials: A free text description
+    -   Attribute type: `Property`.Text
+    -   Optional
+
+-   accuracy: A free text description
+    -   Attribute type: `Property`.Text
+    -   Optional
+
+-   tolerance: A free text description
+    -   Attribute type: `Property`.Text
+    -   Optional
+
+-   emitterExponent: A free text description
+    -   Attribute type: `Property`.Text
+    -   Optional
+
+-   headerError: A free text description
+    -   Attribute type: `Property`.Text
+    -   Optional
+
+-   flowChange: A free text description
+    -   Attribute type: `Property`.Text
+    -   Optional
+
+-   demandCharge: A free text description
+    -   Attribute type: `Property`.Text
+    -   Optional
+
+-   demandCharge: A free text description
+    -   Attribute type: `Property`.Text
+    -   Optional
+
+-   viscosity: A free text description
+    -   Attribute type: `Property`.Text
+    -   Optional
+
+-   unbalanced: A free text description
+    -   Attribute type: `Property`.Text
+    -   Optional
+
+-   checkFrequency: A free text description
+    -   Attribute type: `Property`.Text
+    -   Optional
+
+-   maxCheck: A free text description
+    -   Attribute type: `Property`.Text
+    -   Optional
+
+-   dampLimit: A free text description
+    -   Attribute type: `Property`.Text
+    -   Optional
+
+-   diffusivity: A free text description
+    -   Attribute type: `Property`.Text
+    -   Optional
+
+-   bulkOrder: A free text description
+    -   Attribute type: `Property`.Text
+    -   Optional
+
+-   wallOrder: A free text description
+    -   Attribute type: `Property`.Text
+    -   Optional
+
+-   tankOrder: A free text description
+    -   Attribute type: `Property`.Text
+    -   Optional
+
+-   concentrationLimit: A free text description
+    -   Attribute type: `Property`.Text
+    -   Optional
+
+-   qualityType: A free text description
```

Comment:
Values should be restricted to `NONE`, `CHEM`, `AGE` and `TRACE`.

Reply:
I will update this when updating descriptions.

Reply:
Ok Great! so I need your approval to merge this PR :)

Reply:
Moving to new issue.